README.md

```diff
@@ -245,6 +246,18 @@ By default it will start two containers, one for Faucet server and one for its d
 By default the Faucet allows requests every 24hrs. To disable the timespan check you can pass `FAUCET_TIMESPAN=0` as
 environment variable before starting the script.
 
+### OpenLdap
+
+If the `--ldap` flag is given an OpenLdap service will be started.
+
+- User: `admin`
+- Password: `nevermined`
+
+| Hostname | External Port | Internal URL      | Local URL             | Description                                     |
+| -------- | ------------- | ----------------- | --------------------- | ----------------------------------------------- |
+| `openldap`  | `1389`        | ldap://openldap:389 | ldap://localhost:1389 | [OpenLdap](https://github.com/dinkel/docker-openldap) |
```

Comment:
Is the internal url correct or should the port also be `1389`?

Reply:
yep, that should be the port withing the docker backend network (389). Out of the docker containers, in localhost the port exposed is 1389